packages/react-interactive-tab/package.json

```diff
@@ -1,17 +1,16 @@
 {
   "name": "@fluentui-contrib/react-interactive-tab",
   "version": "0.1.1",
+  "main": "./src/index.js",
+  "typings": "./src/index.d.ts",
   "dependencies": {
     "@fluentui/react-jsx-runtime": "^9.0.29",
+    "@fluentui/react-tabs": "^9.5.0",
```

Comment:
this is intended change (until https://github.com/nrwl/nx/issues/29853 lands) 
- NOTE that this will not introduce duplicates as the version range for sub-package is compatible with suite